app/javascript/app/styles/settings.scss

```diff
@@ -58,13 +57,15 @@ $teal-blue: #045f61;
 $purple: #74356a;
 $light-blue: #035388;
 $error: #d46948;
+$shadow-gray: #e8ecf5;
```

Comment:
We are trying not to add more grays to the settings. Maybe this one is necessary @fannycc ?

Reply:
I need to know where is that grey being use and for what in order to answer that question :)

Reply:
So its:
- Border bottom of the cards
- Border of quantification tags?
- Border of laws and policies cards
- Box shadow of the cards

I'm not sure about the box-shadow but I think we can use one of the grays we already have

Reply:
I used the box-shadow from the UI kit, this color is applied there, and we don't have a similar one.

Reply:
ok right, we can use the lighter one, there is a small difference it's true...I'd say let's keep it if it's being used consistently in shadows. Let me know what you think

Reply:
So to be clear, do you want me to keep this lighter version, `#f1f4fb` right? :thinking:

Reply:
I was putting both options on the table :D so I could know your opinion. We can keep shadows in the same colour they are. If we are using that darker grey consistently across all shadows against white bg I think we can leave it that way. The other option was too light as you said right?

Reply:
I'm not very sharp today, sorry :D The usage of shadows in our app is very inconsistent, `#f1f4fb` shadow we are using lts & ndcs explore page, `#e8ecf5` I applied for updated styles of the cards as it was in UI kit, in many places I see `box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.09); ` and there are also a few similar monsters so it's hard to tell :/ So I'd vote for leaving this as it is now and maybe it's something to identify and review as a separate task, across the whole platform? What do you think?

Reply:
Let's leave it then. But for future style applications let's put the same shadow colour to all of them. If the #f1f4fb works well let's applied that one due to it's already being used in other components. The other grey #e8ecf5 I reckon was in the cards componen in the UI KIT before talking with Alvaro. But we can review it in another moment